content/img/illustrations/illus--Attributeoptions.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Even simpler. Just put:
`<svg xmlns="http://www.w3.org/2000/svg" width="256" height="256"/>`
It will be enough.